code/src/main/kotlin/com/expediagroup/sdk/lodgingconnectivity/graphql/model/response/RawResponse.kt

```diff
@@ -0,0 +1,11 @@
+package com.expediagroup.sdk.lodgingconnectivity.graphql.model.response
+
+open class RawResponse<T>(
+    val data: T,
+    val errors: List<RawResponseError>?
+)
+
+data class RawResponseError(
```

Comment:
Can we just call this `Error`? It could be reusable elsewhere in our codebase.

Reply:
Done!